.github/workflows/build-and-attach-release-runtimes.yml

```
@@ -0,0 +1,69 @@
+name: Build and Attach Runtimes to Releases/RC
+
+on:
+  release:
+    types:
+      - created
+
+env:
+  PROFILE: production
+
+jobs:
+  build_and_upload:
+    strategy:
+      matrix:
+        runtime:
+          - { name: westend, package: westend-runtime, path: polkadot/runtime/westend }
+          - { name: rococo, package: rococo-runtime, path: polkadot/runtime/rococo }
+          - { name: asset-hub-rococo, package: asset-hub-rococo-runtime, path: cumulus/parachains/runtimes/assets/asset-hub-rococo }
+          - { name: asset-hub-westend, package: asset-hub-westend-runtime, path: cumulus/parachains/runtimes/assets/asset-hub-westend }
+          - { name: bridge-hub-rococo, package: bridge-hub-rococo-runtime, path: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo }
+          - { name: contracts-rococo, package: contracts-rococo-runtime, path: cumulus/parachains/runtimes/contracts/contracts-rococo }
+        build_config:
+          # Release build has logging disabled and no dev features
+          - { type: on-chain-release, opts: --features on-chain-release-build } 
+          # Debug build has logging enabled and developer features
+          - { type: dev-debug-build, opts: --features try-runtime }
+
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Build ${{ matrix.runtime.name }} ${{ matrix.build_config.type }}
+      id: srtool_build
+      uses: chevdor/srtool-actions@v0.8.0
+      env:
+        BUILD_OPTS: ${{ matrix.build_config.opts }}
+      with:
+        chain: ${{ matrix.runtime.name }}
+        package: ${{ matrix.runtime.package }}
+        runtime_dir: ${{ matrix.runtime.path }}
+        profile: ${{ env.PROFILE }}
+
+    - name: Build Summary
+      run: |
+        echo '${{ steps.srtool_build.outputs.json }}' | jq . > ${{ matrix.runtime.name }}-srtool-digest.json
```

```suggestion
        echo "${{ steps.srtool_build.outputs.json }}" | jq . > ${{ matrix.runtime.name }}-srtool-digest.json
```

nit but just to be consistent with other non-literal strings in the file.